### PR TITLE
Bump ptlib to 2.10.11

### PR DESCRIPTION
**Pull request by alarcher**
Pre-requisite to Ekiga 4.0.1

### CODE REVIEW

**Comment by pyhalov** [comment]
Dependent packages:
pkg:/communication/conferencing/ekiga
pkg:/library/libopal

File: components/ptlib/Makefile [hunk 1]
```diff
@@ -16,28 +16,32 @@
 include ../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= ptlib
-
-COMPONENT_VERSION= 2.6.7
-COMPONENT_REVISION= 1
+COMPONENT_VERSION= 2.10.11
+COMPONENT_REVISION= 0
```
Comment (AndWac) [inline components/ptlib/Makefile]: Shouldn't COMPONENT_REVISION be removed if COMPONENT_VERSION has changed?

**Comment by alarcher** [comment]
To be merged when #2369 is considered satisfactory.

File: components/ptlib/Makefile (hunk 1 above)
Comment (pyhalov) [reply components/ptlib/Makefile]: Better to remove it

File: components/library/ptlib/patches/ptlib.pc.in.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- ptlib-2.10.11/ptlib.pc.in.orig	2016-07-22 20:44:31.496630089 +0200
```
Comment (pyhalov) [inline components/library/ptlib/patches/ptlib.pc.in.patch]: Is it really needed to add -R${libdir} when usr/lib/libpt.so.$(COMPONENT_VERSION) is delivered?

Comment (alarcher) [reply components/library/ptlib/patches/ptlib.pc.in.patch]: I can remember for now.. I'll have to try again without.

Comment (alarcher) [reply components/library/ptlib/patches/ptlib.pc.in.patch]: Apparently it was useless... Updated, I removed the patch.

**Pull request merged**